docs/autogenerate/Evaluation.adoc

```diff
@@ -0,0 +1,9 @@
+A few things to keep in mind for the evaluation phase:
+
+. At the top-level of your repository, an agent will search for a Nix file to
+   evaluate in the following order: `nix/ci.nix`, `ci.nix` or `default.nix`
+// TODO: link to how pinning is done
+. `NIX_PATH` will always be empty. Make sure you pin nixpkgs and not rely on `<nixpkgs>`
```

Comment:
```suggestion
. `NIX_PATH` will be empty. Make sure you pin nixpkgs and not rely on `<nixpkgs>`
```
"Always" sounds like a promise. We will support this (but no need to mention that here)